nearuplib/localnet.py

```diff
@@ -30,7 +30,7 @@ def run(binary_path, home, num_nodes, num_shards, override, verbose=True, intera
         with each other on localhost, producing blocks on top
         of a genesis block generated locally.
         """)))
-        print('')
```

Comment:
Can't we just use a newline in line 31 instead of this print?

Reply:
oh true...